src/ibex_bluesky_core/callbacks/document_logger.py

```diff
@@ -0,0 +1,40 @@
+"""Logs all documents that the BlueSky run engine creates via a callback."""
```

Comment:
I think the new `callbacks` module should have an `__init__.py` to be importable